config/initializers/unify_azure.rb

```diff
@@ -0,0 +1,81 @@
+require 'fog/azure/compute'
+require 'fog/azure/models/compute/images'
+require 'fog/azure/models/compute/server'
+require 'fog/azure/models/compute/servers'
+
+class Fog::Compute::Azure::Server
+  def stop
+    raise Atmosphere::UnsupportedException, 'Azure des not support stop action'
+  end
+  def suspend
+    shutdown
+  end
+  def pause
+    raise Atmosphere::UnsupportedException, 'Azure des not support pause action'
+  end
+  def id
+    identity
+  end
+
+  def flavor
+    {'id' => flavor_id}
+  end
+
+  def image_id
+    attributes[:image]
+  end
+
+  def created
+    vm = Atmosphere::VirtualMachine.find_by(id_at_site: id)
+    vm ? vm.created_at : (Atmosphere.childhood_age - 1).seconds.ago
+  end
+
+  def task_state
+    nil
+  end
+
+  def flavor_id
+    attributes[:vm_size]
+  end
+end
+
+class Fog::Compute::Azure::Servers
+  def destroy(id_at_site)
+    # get requires both identity and cloud service name params
+    # in our case id == cloud service name
+    server = all.find{|s| s.name == id_at_site}
```

Comment:
Prefer `detect` over `find`.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.